engine/src/cubos/engine/plugins/imgui.cpp

```diff
@@ -0,0 +1,29 @@
+#include <cubos/engine/plugins/imgui.hpp>
+#include <cubos/engine/plugins/window.hpp>
+#include <cubos/core/ui/imgui.hpp>
+
+static void initializeImGui(const cubos::core::io::Window& window)
+{
+    cubos::core::ui::initialize(window);
+}
+
+static void beginImGuiFrame()
+{
+    cubos::core::ui::beginFrame();
+}
+
+static void endImGuiFrame()
+{
+    cubos::core::ui::endFrame();
+}
```

Comment:
You can just pass `cubos::core::ui::initialize`, `cubos::core::ui::beginFrame` and `cubos::core::ui::endFrame()` directly to `addSystem`, since they have a valid system signature.

Reply:
Not sure how I could call pass the window resource to `cubos::core::ui::initialize`

Reply:
Okay true, that one does need to stay like it is right now, but you can do it for begin and end frame.

Reply:
Not possible because `cubos::core::ui::endFrame` receives a `cubos::core::gl::Framebuffer` argument.
In order to maintain consistency, I'm letting them as they are.